packages/vuetify/src/components/VOverlay/_variables.scss

```diff
@@ -1,3 +1,3 @@
 @import '../../styles/styles.sass';
 
-$overlay-transition: $primary-transition, z-index 1ms !default;
+$overlay-opacity: 32%;
```

Comment:
in the future this could probably use disabled opacity which is .38

Reply:
material.io uses 32% for scrims